cypress-io/cypress
# chore: remove cloud connect button from app

- Closes #25926 

### Additional details
Currently when a user is not connected to the cloud, the Specs list page will show links when hovering over the empty states in the "Latest Runs" and the "Average Duration" columns (the button that says "connect"). This change removes this functionality as it is no longer needed.


### Steps to test
- Load the app via `yarn workspace @packages/app dev`
- Make sure you are logged out of the cloud
- Hover over any of the `Latest Runs` and `Average Duration` cells

### How has the user experience changed?

https://user-images.githubusercontent.com/3605268/222523025-1c775695-fca4-4804-83d6-c761881dde92.mp4

### PR Tasks
